tensorflow/compiler/xla/service/gpu/gemm_algorithm_picker.cc

```diff
@@ -302,6 +302,8 @@ StatusOr<std::optional<se::blas::AlgorithmType>> DoGemmAutotune(
                                        *gemm->operand(has_matrix_bias ? 3 : 2),
                                        autotune_config, rng_state));
     }
+    se::DeviceMemoryBase a_scale_buffer, b_scale_buffer, c_scale_buffer,
+        d_scale_buffer, d_amax_buffer;
```

Comment:
Do you need to set these for FP8 matmuls?

Reply:
Autotuning is only relevant for architectures older than Ampere. When the buffers for the scaling factors aren’t set, the factors are assumed to be one.